### DISCUSSION TITLE
Any way to do simple arithmetic in the `run-name`?

### DISCUSSION THREAD

**Discussion by paulmr** [community/community#37193] [Actions]
Is there any way that any one can think of to include simple arithmetic in the `run-name` to give a dynamic name to the current run of a workflow? For historical reasons our deployment tool uses a build number that is the GHA run number + some fixed quantity, and we would like to include this deployment build number in the run name for easy reference. When I saw that it was possible to include expressions in the `run-name` I thought that would be the solution, but it looks like these expressions just have boolean operators and some simple boolean functions. It does make sense not to be able to include too much logic in these expressions, but is there any way to do simple arithmetic? With the existence of string functions like `join()` in mind, what I am imagining is, for example, a function like `numericAdd()` which would parse the arguments as integers and then add them together:

```yaml
name: Example workflow
run-name: Deploy build ${{ numericAdd(github.run_number, 546) }}
```

Another alternative implementation we thought of was if there was some way to attach metadata to a run with an action, like a tag or a label with a value, but I can’t see any metadata like that.

**Comment by ossanna16** [comment]
Hi there @paulmr and welcome to our community! Thank you for asking a great question 🙂 

While you are waiting for a reply, here are a few things you can do:

* To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
* To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
